langkit/whylogs/example_utils/guardrails_llm_schema.py

```diff
@@ -0,0 +1,145 @@
+"""
+This is an auxiliary script used in LangKit's examples: Safeguarding and Monitoring LLM applications.
+In this file, we are going to define a whylogs logger that will be used for  a) content moderation, b) message auditing,
+and c) observability. While logging, we will define validators that will check for toxic content and forbidden regex patterns.
+Whenever a condition fails to be met, an action is triggered that will update the moderation queue  with the relevant flags for the related message id.
+
+The logger will also generate statistical profiles every 5 minutes and send them to WhyLabs for observability.
+
+"""
+from langkit import regexes
+from langkit import sentiment
+from langkit import textstat
+from langkit import themes
+from langkit import toxicity
+from whylogs.core.relations import Predicate
+from typing import Any
+from whylogs.core.metrics.condition_count_metric import Condition
+import whylogs as why
+from whylogs.experimental.core.metrics.udf_metric import udf_metric_schema
+from whylogs.core.validators import ConditionValidator
+from whylogs.core.metrics import MetricConfig
+from typing import TypedDict
```

Comment:
any reason we're not doing import grouping here?

Reply:
the only reason is because, since I'm logging prompts and responses separately, importing `input_output` wouldn't work, and it might make users curious as to why it's not in the dashboard.